nvm.sh

```diff
@@ -2582,6 +2585,14 @@ nvm() {
             NVM_UPGRADE_NPM=1
             shift
           ;;
+          --default)
+            ALIAS='default'
+            shift
+          ;;
+          --alias=*)
+            ALIAS="${1##--alias=}"
```

Comment:
Can we ensure that both `--default` and `--alias` are not provided, preferably by erroring out? (with a test)